src/helm.exec.ts

```diff
@@ -570,6 +572,32 @@ export async function helmExecAsync(args: string): Promise<ShellResult | undefin
     return await sh.exec(cmd);
 }
 
+const HELM_BINARY: ExternalBinary = {
+    binBaseName: 'helm',
+    configKeyName: 'helm',
+    displayName: 'Helm',
+    offersInstall: true,
+};
+
+const HELM_CONTEXT: Context = {
+    host: host,
+    fs: shellfs,
+    shell: shell,
+    pathfinder: undefined,
+    binary: HELM_BINARY,
+    status: undefined,
+};
+
+export async function helmInvokeCommand(command: string): Promise<ExecResult> {
```

Comment:
This function is never called. Can it be deleted?

Reply:
It could be, but many of the other Helm functions will be ported over to use it (I think I must have started doing that and then backed them out because I was doing too much at once).